data/announcements/scheduled.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# For an example of the format, see commented structure below.
+#
+# 🛈  Changes require approval from @kubernetes/steering-committee
+
+# This is just an example. For the actual announcements, see
+# announcements.yaml
```

Comment:
Is there a reason to have this file (template and docs) and a separate file for the announcement data?
What do you think about combining the instructions and template with the data?

Reply:
> What do you think about combining the instructions and template with the data?

Agree, especially if the data has to be a in a single list in a single file.

Reply:
It is possible to have multiple files, though, I find it easier if the data is in one place or presented as a single view.